src/templates/directoryListing.hbs

```diff
@@ -1,31 +1,11 @@
 <html>
-  <head>
-    <title>Index of {{pathname}}</title>
-    <meta name='viewport' content='width=device-width, initial-scale=1.0' />
-    <meta charset='utf-8' />
-    <style type='text/css'>
-      td { padding-right: 16px; text-align: right; font-family: monospace }
-      td:nth-of-type(1) { text-align: left; overflow-wrap: anywhere }
-      td:nth-of-type(3) { white-space: nowrap } th { text-align: left; } @media
-      (prefers-color-scheme: dark) { body { color: white; background-color:
-      #1c1b22; } a { color: #3391ff; } a:visited { color: #C63B65; } }
-    </style>
-  </head>
+  <head><title>Index of {{pathname}}</title></head>
   <body>
-    <h1>Index of {{pathname}}</h1>
-    <table>
-      <tr>
-        <th>Filename</th>
-        <th>Modified</th>
-        <th>Size</th>
-      </tr>
+    <h1>Index of {{pathname}}</h1><hr /><pre><a href='../'>../</a><br />
       {{#each entries}}
-        <tr>
-          <td><a href='{{href}}'>{{name}}</a></td>
-          <td>{{lastModified}}</td>
-          <td>{{size}}</td>
-        </tr>
+        <a href='{{href}}'>{{name}}</a>
+        {{lastModified}}
+        {{size}}<br />
```

Comment:
```suggestion
        {{size}}
```

`br` isn't necessary inside a `pre`.

Side-note: self-closing tags don't exist in HTML